nixos/lib/build-vms.nix

```diff
@@ -6,7 +6,8 @@
   # Nixpkgs, for qemu, lib and more
 , pkgs
 , # NixOS configuration to add to the VMs
-  extraConfigurations ? []
+  extraConfigurations ? let e = builtins.getEnv "NIXOS_EXTRA_MODULE_PATH";
+                        in if e == "" then [] else [(import e)]
```

Comment:
Hrm, a bit ugly to proliferate this code around. Are there nicer ways to accomplish this?

Reply:
Maybe we can just not put it most places, and then see what breaks :D